### DISCUSSION TITLE
Convert the Argo Rollouts CLI to standalone executable

### DISCUSSION THREAD

**Discussion by kostis-codefresh** [argoproj/argo-rollouts#1089] [Ideas]
Currently the Argo Rollouts CLI is documented to be a kubectl plugin. However I have successfully used the CLI on its own for most needed actions (watching a rollout, promoting a canary etc) without the need of kubectl.

Unless I am missing something the Argo Rollouts CLI is self-contained.

Would it make sense for consistency reasons to just document it as a standalone executable? The other Argo projects seems to do the same. The Argo Workflows and ArgoCD CLIs are normal CLIs and not kubectl plugins.

**Comment by dsbos** [comment]
It would also be helpful the documentation explained what exactly is needed for the CLI to actually function as a `kubectl` plug-in. (When I installed the CLI (per recent instructions), my `kubectl` command did not recognize the `kubectl argo ...` syntax.
